src/pybind/mgr/cephadm/module.py

```diff
@@ -2193,6 +2197,27 @@ def _refresh_configs(self):
                 self._create_daemon(dd.daemon_type, dd.daemon_id,
                                     dd.hostname, reconfig=True)
 
+        # make sure the dashboard [does not] references grafana
+        try:
+            current_url = self.remote('dashboard', 'get_option',
```

Comment:
This line seems to cause trouble for me when trying to run the code in this PR using `vstart` with `--cephadm`. vstart will reproducibly hang when setting `cephadm` as the orchestrator backend. There'll be a segfault in the mgr logs.

If I replace the line with 

```
current_url = self.get_module_option_ex('dashboard', 'GRAFANA_API_URL')
```

it'll work. I've also checked that it returns the configured URL for Grafana.